### ISSUE TITLE
CDN 403 error

### ISSUE THREAD

**Issue by hoodmane** [iodide-project/pyodide#952]
There's a permission issue. I get a 403 error when I try to visit the dev console:
https://cdn.jsdelivr.net/pyodide/dev/full/console.html

**Comment by dalcde** [comment]
I think the recommended path is https://pyodide-cdn2.iodide.io/dev/full/console.html which is broken for a different reason

**Comment by hoodmane** [comment]
The latter works for me.

**Comment by dalcde** [comment]
I get the error
```
The resource from “https://cdn.jsdelivr.net/pyodide/v0.16.0/full/pyodide.asm.data.js” was blocked due to MIME type (“text/plain”) mismatch 
```
In any case, it is trying to load from v0.16.0 instead of dev. This should be fixed by #953
